packages/fetch-and-compile/package-lock.json

```diff
@@ -0,0 +1,876 @@
+{
```

Comment:
This file doesn't belong in a yarn managed project.

Reply:
Makes sense, I deleted it. Thanks.